doc/elements.md

```diff
@@ -358,6 +358,7 @@ Attribute | Type | Default | description
 `file-name` | string | - | The name of this file; will be used to store this file in the `_files` submitted answer
 `ace-mode` | string | None | Specifies an Ace editor mode to enable things like intelligent code indenting and syntax highlighting; see the full list of modes [here](https://github.com/ajaxorg/ace/tree/master/lib/ace/mode).
 `ace-theme` | string | `ace/theme/chrome` | Specifies an Ace editor theme; see the full list of themes [here](https://github.com/ajaxorg/ace/tree/master/lib/ace/theme).
+`source-file-name` | string | None | Name of the file with content to be displayed inside the text editor (instead - or in addition to - writing the text between the open/closing tags).
```

Comment:
What's the use case for having starting contents come from both a file and the element contents? Not opposed to this necessarily, just curious.

Reply:
I didn't think I could remove the option to add content from inside the element, so I just added the option to add content from a source file as well.

In my case, I would just use one or the other, but I think keeping both adds more flexibility?

Reply:
I don't take issue with that, it's the appending behavior that's potentially confusing. I feel like it should be one or the other, probably enforced in code.

Reply:
okay, I will make it be one or the other. How do I go about `pl-code`. Do I start another branch and do another PR?

Reply:
also, since it is one or the other, I am assuming the file content will override the element content when someone uses both? or should it throw an error?

Reply:
Yep, just open a new PR. @mwest1066 what do you think about throwing an error? I think enforcing that it's only one just so behavior isn't silently wrong is probably best.

Reply:
Yeah, new PR for the `pl-code` fix.

I think throwing an error if both are provided is a good idea, to save anyone from surprises. For the file it's easy to tell whether it's provided (do we have the `source-file-name` attribute or not). For the inline text, I think it's a bit harder to tell whether it's provided. I think we need to check whether there is text, and if so, whether the text is just whitespace?